config.md

```diff
@@ -95,6 +95,7 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 **`process`** (object, required) configures the container process.
 
 * **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process, defaults to false.
+* **`initialConsoleSize`** (box, optional) specifies the initial console size of the terminal if attached.
```

Comment:
@hqhq - @laijs explicitly said this was needed on Linux, so I don't understand your comment? As for `initial`, this is important - it's the how the console is initialised for the console start, so I don't agree with your other suggestion.